### DISCUSSION TITLE
Suppression A320 dans MSFS

### DISCUSSION THREAD

**Discussion by looping77** [flybywiresim/aircraft#5679] [Q&A] [answered]
Bonjour, j'ai un soucis avec la tablette dans mon A320 qui reste blanche ainsi que mes feux d'atterrissage qui ne sont plus fonctionnelles depuis la mise à jour MSFS. j'ai vue la solution qui consiste à supprimer dans le gestionnaire de contenue l'A320 de FBW mais malheureusement quand je sélectionne la ligne, je n'ai pas de bouton pour supprimer qui apparait donc il m'est impossible de le supprimer. avez-vous une solution à ce problème.
Merci pour votre aide.

Bernard

**Comment by hotshotp** [comment, marked as answer]
update a32nx to latest dev/stable. for more help ask in the support channel in the discord

**Comment by MicroCire** [comment]
Bonjour Bernard, pour essayer de solutionner ce problème 2 choses: enlever a320.... de community et aller dans packages et supprimer les fichiers s'y trouvant sauf le sous répertoire work qui contient les paramètres de vos contrôleurs. Refaire une install avec installateur (version 2 je ccrois a l'heure actuelle). Chez moi le probleme a été résolu. Je parle pour la version dev

Hello Bernard, to try to solve this problem 2 things: remove a320 .... from community and go to packages and delete the files located there except the work subdirectory which contains the parameters of your controllers. Redo an install with installer (version 2 I believe at the moment). At home the problem has been solved.
